content/applications/marketing/marketing_automation/campaign_templates/double_optin.rst

```diff
@@ -0,0 +1,153 @@
+
+Upon creation of the campaign, the campaign form loads with a new preconfigured campaign.
+
+The :guilabel:`Target` and :guilabel:`Filter` configuration of the campaign is as follows:
+
+- :guilabel:`Name`: `Double Opt-in`
+- :guilabel:`Responsible`: The user who created the campaign.
+- :guilabel:`Target`: :guilabel:`Mailing Contact`
+- :guilabel:`Unicity based on`: :guilabel:`Email (Mailing Contact)`
+- :guilabel:`Filter`:
+
+  - :guilabel:`Email` :guilabel:`is set`
+  - :guilabel:`Blacklist` :guilabel:`is not` :guilabel:`set`
+  - :guilabel:`Mailing lists` :guilabel:`contains` `Newsletter`
+
+.. important::
+   The :guilabel:`Target` model of the campaign should not be modified. Changing the
+   :guilabel:`Target` model with activities in the :guilabel:`Workflow` invalidates the existing
+   activities in the :guilabel:`Workflow`.
+
+   The *Double Opt-in* campaign template is intended to only use the :guilabel:`Mailing Contact`
+   model.
+
+The campaign loads two activities in the :guilabel:`Workflow` section of the campaign: an email
+activity with a child server action activity that triggers *on click*.
+
+By default, the `Confirmation` email activity is set to trigger :guilabel:`1 Hours` after the
+beginning of the workflow. In other words, the email is sent 1 hour after a new contact is added to
+the *Newsletter* mailing list.
+
+The email activity uses the preconfigured *Confirmation* email template, which contains a button for
+the contact to click to confirm their consent.
+
+To modify the email template, select the :icon:`fa-envelope-o` :guilabel:`Templates` smart button at
+the top of the campaign form. Then, in the list of templates, select the `Confirmation` email
+template.
+
+Be sure to personalize the contents of the email template; however, it is recommended to keep the
+contents of double opt-in confirmation emails short and to-the-point.
+
+The default confirmation button links directly to the database's website homepage. Click on the
+button to edit the button text and URL.
+
+.. tip::
+   To provide a streamlined user experience for the contact, consider :doc:`creating a page on the
+   website <../../../websites/website/pages>` that expresses gratitude to the contact for
+   confirming their subscription to the mailing list. Add the link to that page in the URL of the
+   confirmation button.
+
+.. important::
+   The email template should only include a single call to action link for confirmation, other than
+   an unsubscribe link.
+
+   Any click on a link or button included in the confirmation email, besides the unsubscribe button,
+   triggers the *Add to list* server action.
+
+   The child activity *Add to list* server action's *On click* trigger cannot differentiate between
+   multiple URLs in an email, besides the `/unsubscribe_from_list` unsubscribe button that is
+   included in any one of the footer blocks.
+
+The `Add to list` server action child activity is set to trigger immediately after a click in the
```

Comment:
```suggestion
The :guilabel:`Add to list` server action child activity is set to trigger immediately after a click in the
```
suggesting a GUI here because, at this point, the reader would be back on the campaign form page, wherein this would *actually be seen* on the screen